hackathon_site/hardware/views.py

```diff
@@ -62,9 +69,31 @@ class IncidentListView(mixins.ListModelMixin, generics.GenericAPIView):
     filter_backends = (filters.DjangoFilterBackend, SearchFilter)
     filterset_class = IncidentFilter
 
+    def get_serializer_class(self):
```

Comment:
tbh I don't remember why we implemented this in the Order view. We should have just had an if/elif statement on `self.request.method` in here and returned the appropriate one, rather than having the map and try/except.